nbproject/genfiles.properties

```diff
@@ -8,4 +8,4 @@ nbproject/build-impl.xml.script.CRC32=b72261eb
 nbproject/build-impl.xml.stylesheet.CRC32=0f381476@2.47.1
 nbproject/platform.xml.data.CRC32=2e7964b0
 nbproject/platform.xml.script.CRC32=6dcbd131
-nbproject/platform.xml.stylesheet.CRC32=4e1f53d4@2.62.1
+nbproject/platform.xml.stylesheet.CRC32=45ddf0e0@2.72.1
```

Comment:
This is the first time this file has been checked in in three years. Do you guys have a different .gitignore perhaps? Should we care about having these CRCs up to date, e.g., does NetBeans check them and do more work during builds if they are out of date?